patches/activate-cross-signing-and-secure-storage-react/matrix-react-sdk+3.63.0.patch

```diff
@@ -126,7 +126,87 @@ index 2e6e6fb..dae664a 100644
  
  interface IState {
      loading: boolean;
-@@ -460,6 +461,12 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
+@@ -244,25 +245,27 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
+             if (MatrixClientPeg.get().getKeyBackupEnabled()) {
+                 statusDescription = <p>✅ {_t("This session is backing up your keys. ")}</p>;
+             } else {
+-                statusDescription = (
+-                    <>
+-                        <p>
+-                            {_t(
+-                                "This session is <b>not backing up your keys</b>, " +
+-                                    "but you do have an existing backup you can restore from " +
+-                                    "and add to going forward.",
+-                                {},
+-                                { b: (sub) => <b>{sub}</b> },
+-                            )}
+-                        </p>
+-                        <p>
+-                            {_t(
+-                                "Connect this session to key backup before signing out to avoid " +
+-                                    "losing any keys that may only be on this session.",
+-                            )}
+-                        </p>
+-                    </>
+-                );
++                // :TCHAP remove unecessary complex text
++                // statusDescription = (
++                //     <>
++                //         <p>
++                //             {_t(
++                //                 "This session is <b>not backing up your keys</b>, " +
++                //                     "but you do have an existing backup you can restore from " +
++                //                     "and add to going forward.",
++                //                 {},
++                //                 { b: (sub) => <b>{sub}</b> },
++                //             )}
++                //         </p>
++                //         <p>
++                //             {_t(
++                //                 "Connect this session to key backup before signing out to avoid " +
++                //                     "losing any keys that may only be on this session.",
++                //             )}
++                //         </p>
++                //     </>
++                // );
++                // :end TCHAP
+                 restoreButtonCaption = _t("Connect this session to Key Backup");
+             }
+ 
+@@ -420,18 +423,20 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
+                 );
+             }
+         } else {
+-            statusDescription = (
```

Comment:
I would rather keep this text for the beta phase to be able to understand easily what is going on.